activesupport/lib/active_support/core_ext/enumerable.rb

```diff
@@ -114,11 +114,15 @@ def sum(identity = nil)
 # just calling the compat method in the first place.
 if Array.instance_methods(false).include?(:sum) && !(%w[a].sum rescue false)
   class Array
-    remove_method :sum
+    alias :orig_sum :sum
```

Comment:
Should `orig_sum` method be public API? (this implementation seems to make it so). If not, you can mark it as private.